src/common/models/Projection.ts

```diff
@@ -454,6 +493,16 @@ export class Projection {
     return null;
   }
 
+  private divideICUDataWithNulls(data: HospitalResourceUtilization | null) {
```

Comment:
Not wild about this method name.  I think I'd rename it `calcIcuCapacityUsage` or something, and maybe add a comment about the null-handling if you think it's useful.